### ISSUE TITLE
missing type link when run in CI

### ISSUE THREAD

**Issue by jan-xyz** [princjef/gomarkdoc#114]
When I locally generate the markdown it seems to differ from the output that is generated when GitHub actions runs the CLI

```
internal/plugins/appstore/README.md - (expected) (actual)
19 | 19 |   
20 | 20 |   <a name="App"></a>
21 | 21 |   ## type [App](<
https://github.com/jan-xyz/dotfiles/blob/main/internal/plugins/appstore/plugin.go#L14-L17
>)
Error: output does not match current files. Did you forget to run gomarkdoc?
```
(The expected here is `## type App`)

I run it with 
`github.com/princjef/gomarkdoc/cmd/gomarkdoc --check --output "{{.Dir}}/README.md" --format github ./internal/...` to check it and with `go run github.com/princjef/gomarkdoc/cmd/gomarkdoc --output "{{.Dir}}/README.md"  --format github ./internal/...` to generate the markdown.

**Comment by sunglim** [comment]
The same in my workflow. It failed in only in workflow. When I run `gomarkdoc ./...` on my local, it is fine.
 - [workflow](https://github.com/sunglim/indicator/blob/workflowtest/.github/workflows/doc.yml)
 - [result](https://github.com/sunglim/indicator/actions/runs/9927195150/job/27421849961)
 
```
 <a name="Abs"></a>
-## func [Abs](<https://github.com/cinar/indicator/blob/master/helper/abs.go#L15>)
+## func Abs
```

**Comment by sunglim** [comment]
This issue is a duplicate of https://github.com/princjef/gomarkdoc/issues/84

**Comment by jan-xyz** [comment]
Thanks, this solved it! https://github.com/jan-xyz/Dotfiles/pull/115

**Comment by sunglim** [comment]
> Thanks, this solved it! [jan-xyz/Dotfiles#115](https://github.com/jan-xyz/Dotfiles/pull/115)

I'm happy to help.  I could get a hint by using -vv option. it prints INFO log ;)
